### PR TITLE
add FAQ page

### LINKED ISSUE #197: Add FAQ section to documentation
For now I think the FAQ should unify the project components rather than hosting separate FAQs in each repository.

### PR DESCRIPTION
**Pull request by wholmgren**
closes #197. needs lots of work.

### CODE REVIEW

**Comment by lboeman** [comment]
Added a couple questions and did some styling here: https://lboeman.github.io/faq/ I think it helps break up the Q/As. If you approve, I'll just push to your branch.

**Comment by wholmgren** [comment]
I like the styling. Go ahead and push to my branch.

**Review by lboeman — APPROVED** [review]

**Comment by wholmgren** [comment]
@lboeman can you proof the additions and merge if you're happy?

File: faq.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+  A. The [API documentation](https://api.solarforecastarbiter.org/) contains a
+  number of request and response examples. For examples using the Python API
+  wrapper, please see the [data_upload_download
+  notebook](https://github.com/SolarArbiter/workshop/blob/master/data_upload_download.ipynb)
+  in our [workshop repository](https://github.com/SolarArbiter/workshop). You
+  can actually run that notebook using
+  [MyBinder](https://mybinder.org/v2/gh/SolarArbiter/workshop/master) if you
+  want a very quick start.
+
+- Q. I can access the dashboard using my credentials but not the API. Why not?
```
Comment (lboeman) [inline faq.md:116]:
```suggestion
- Q. I'm having trouble connecting to the API or requesting an access token securely. What gives?
```
Maybe we should reference this older issue when this first cropped up? https://github.com/SolarArbiter/solarforecastarbiter-core/issues/474#issuecomment-638601890 The more recent issue was in requesting an access token from auth0, but it looks like it has cropped up before as well. I'll update the answer in another comment.

File: faq.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+  A. If you're behind a firewall, there is a good chance that is causing
+  problems with reaching our identity provider, Auth0. If you're using the SFA
+  core library or the python ``requests`` library, you can get around this by
+  copying the PEM chain certificate for
+  https://solarforecastarbiter.auth0.com/oauth/token to your machine and setting
+  the ``REQUESTS_CA_BUNDLE`` environment variable to the path to the
+  certificate.
```
Comment (lboeman) [inline faq.md:124]:
```suggestion
  A. If you're behind a firewall, there is a good chance that is causing
  problems with verifying the SFA or Auth0 certificates. If you're using the SFA
  core library or the python ``requests`` library, you can get around this by
  copying the appropriate PEM chain certificate to your machine and setting
  the ``REQUESTS_CA_BUNDLE`` environment variable to the path to the
  certificate. These certs can typically be found by visiting the offending url
  in your browser and clicking the lock on the left side of the search bar.
  6See https://github.com/SolarArbiter/solarforecastarbiter-core/issues/474 for
  more context.
```

**Review by lboeman — APPROVED** [review]
Looks good. I had some minor changes to the part that I had changed. but that's about it.

**Pull request merged**